manifest.json

```diff
@@ -0,0 +1,156 @@
+{
+  "manifest_version": "0.2",
+  "name": "signoz",
+  "display_name": "SigNoz MCP Server",
+  "version": "0.1.0",
```

Comment:
the current signoz mcp server version is v0.0.4, do you think we should keep it aligned.